nilearn/plotting/html_surface.py

```diff
@@ -252,10 +354,22 @@ def view_surf(surf_mesh, surf_map=None, bg_map=None, threshold=None,
         .thickness, .area, .curv, .sulc, .annot, .label) or
         a Numpy array
 
-    bg_map : Surface data, optional
+    bg_map : Surface data object (to be defined), optional
```

Comment:
same here

Reply:
Sorry, what do you mean exactly? :blush: